packages/core/src/components/dropdown-menu/dropdown-menu.global.scss

```diff
@@ -0,0 +1,19 @@
+dso-dropdown-menu {
+  @include dropdown-menu_children();
+
+  &[dropdown-align="right"] {
+    @include dropdown-menu_align-right();
+
+    .dso-dropdown-menu {
+      @include dropdown-menu_right();
```

Comment:
Ik weet niet precies wat de gedachte achter de constructie is, maar 'n mixin die `align-right()` en 'n andere mixin die `right()` heet is natuurlijk 'n onduidelijk verhaal..

Daarnaast was de scope van de feature anders: Het ging over het alignen van de dropdown, niet over het positioneren van de knop:

![image](https://user-images.githubusercontent.com/9696261/126793906-f81fe931-225f-46fb-a370-a01776fc27b2.png)

![image](https://user-images.githubusercontent.com/9696261/126794026-b590f969-f993-4de7-b036-c2b621a3dca3.png)